cli/src/cmd/pkg/validate.go

```diff
@@ -0,0 +1,84 @@
+package pkg
+
+import (
+	"cli/core"
+	"cli/util"
+	"path"
+	"strings"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrNoConfigImage            = errors.New("config file missing field 'image'")
+	ErrNoPackages               = errors.New("no packages or custom packages specified")
+	ErrUndefinedProfilePackages = errors.New("packages in profile not in any of packages, custom-packages, or command-line custom-packages")
+)
+
+func validate(cmd *cobra.Command, config *core.Config) error {
+	customPackagePaths, err := cmd.Flags().GetStringSlice("custom-path")
```

Comment:
Good call! I added the function in [68bfbf0](https://github.com/openhie/package-starter-kit/pull/19/commits/68bfbf01becaaffc87137308b8b67a7c27e3267e) and tested it in [8f5d4cb](https://github.com/openhie/package-starter-kit/pull/19/commits/8f5d4cb03639a144f8e74d795eae5b07fb18e763)

Reply:
On second thought, we actually shouldn't validate this at the cli-level. Validation of packages happens at the
instant.ts level, it shoudn't be done here because custom package repo
names don't have to coincide with their id in their respective
package-metadata.json files

Reply:
It shouldn't fail in that case, yes, and I have added a test for that. But it will fail you're passing through 
```Go
config.Packages = []string{}
config.CustomPackages = []core.CustomPackage{}
```
without expecting an error. Look at where the config is populated, and where the hookFunc is executed, and what the hookFunc does for that test case to the config.

Check the test I added in [9f72819](https://github.com/openhie/package-starter-kit/pull/19/commits/9f72819cfb3260fe7a474da300dfe9a45d4d2adc)

Reply:
cool got u 👍🏼